CHANGELOG.md

```diff
@@ -2,13 +2,9 @@
 
 ## 1.1.0
 
-2019-04-19
```

Comment:
I removed dates for consistency with other repos and because revision history is more informative for this purpose.  If the previous version was 1.1.0-SNAPSHOT, maybe this bugfix needn't be 1.1.1… 1.1.0 was published to npmjs.org, but not merged to upstream/master or released here.  I think we could go either way, but I'm treating this as more authoritative than npm, resting assured that they'll become consistent via the pipeline soon, and/or myuw-help-link will just get subsumed into myuw-help (like myuw-drawer and myuw-drawer-link).